hdbt.theme

```diff
@@ -1317,6 +1317,13 @@ function hdbt_preprocess_tpr_unit(&$variables): void {
   }
   $entity = $variables['entity'];
 
+  // Add unit publish state to variable.
+  $variables['unit_published'] = TRUE;
```

Comment:
This variable could be more generic. Like `$variables['content_entity_published']`

Reply:
Made it generic 🦖